benchmark/BENCHMARK.md

```diff
@@ -35,9 +35,18 @@ Some scenarios (`CreateDeleteClients`, and `CrawlUsers`, `CreateRealms`, `Create
    * click save
 * Change to the tab `Sevice Account Roles`
    * select for `realm-management` in the `Client Roles` listbox
-   * assign the roles `manage-clients` and `view-users`
+   * assign the roles, based on the below role-mapping table for the respective load simulation scenario
 * the client secret to be passed to the tests can be copied from the `Credentials` tab
 
+| Scenario Name            |       Assigned Roles       |
```

Comment:
There are also the Create-without-Delete scenarios. What about adding them into the same line with their Delete-variants?

Reply:
sure thing, I will add those as well

Reply:
done